### PR TITLE
Portfolio desktop branch

### PR DESCRIPTION
**Pull request by H-Ramalan**
•	Set up linters files: Configured ESLint, Hint, and Stylelint to ensure consistent code formatting and catch any potential syntax errors.
•	Added markup with styling: I added semantic HTML markup and CSS styling to improve the visual appeal and accessibility of the project.
•	Used Gitflow: Following the Gitflow branching model to ensure a structured workflow for the project and to make collaboration with other developers more efficient.
Kindly help review and let me know if any further modifications are needed.

Thank you for your review.

### CODE REVIEW

**Review by AbelG101 — COMMENTED** [review]
# Hi STUDENT_MENTION 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- You have submitted a project, that doesn't have a running linter check. So please make sure you configured the appropriate linters which [you can find here](https://github.com/microverseinc/linters-config/tree/master/html-css)
  firefox_jggpKyxfkV

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: style.css [hunk 1]
```diff
@@ -0,0 +1,920 @@
...
+.nav-link-container {
+  display: flex;
+  align-items: center;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  font-weight: 600;
+  line-height: 20px;
+}
+
+@media screen and (max-width: 768px) {
```
Comment (divinecharlotte) [inline style.css]: - [ ] kindly consider enclosing all the desktop version CSS in a desktop media query like  this`@media screen and (min-width: 768px) {}` as you did here on the mobile version  to make your code more organized and make sure that this requirement is fully filled 
> Use media queries in your CSS file to make your desktop version look like the template you selected `(Desktop: >=768px).`

File: style.css [hunk 2]
```diff
@@ -0,0 +1,920 @@
...
+    text-align: center;
+  }
+}
+
+li.languages {
+  list-style: none;
+}
+
+li.languages::before {
+  color: orange;
```
Comment (divinecharlotte) [inline style.css]: - [ ] Kindly consider sticking to the Figma design as much as possible by giving these icons the same color as the one given in the design the reason is to make your project an exact duplicate of the given one check the screenshots below to see the difference 
- your design 👇 
Screen Shot 2023-05-12 at 13 50 22



- Figma design 👇 
Screen Shot 2023-05-12 at 13 50 47

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @H-Ramalan    , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 
-Descriptive commit messages👍
- linters are passing 👍 




 

## Required Changes ♻️

_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html
Comment (Efiamotu-1) [inline index.html:1]: [OPTIONAL]
- [ ] please it is a project requirement that your mobile version stays correct as much as your desktop version however from the screen below of your header and about section it is visible that the background images are not displaying as they should on mobile. i kundly suggest that you fix this design error :+1: :sparkles: 
- your designs header & about section
bg-h
bg-ab

Comment (Efiamotu-1) [inline index.html:1]: [OPTIONAL]
- [ ] you currently do not have enough spaciing between your content and the border of your toolbar section, please kindly add the necessary spacing on desktop version so it matches correctly the figma design as shown below :+1: 
- [ ] Also you making use of the word `my logo` as your logo, but you can alternatively put in your name which is more descriptive and professional to use :+1: :grin: :sparkles: 
- figma design
logo
- your design
logo

**Review by Efiamotu-1 — APPROVED** [review]
Hi @H-Ramalan , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

giphy


## Highlights :dart: 
* Correct GitHub flow :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good PR summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**